### ISSUE TITLE
Ensure that collated subdirectories are archived deterministically

### ISSUE THREAD

**Issue by Donaim** [cfe-lab/MiCall#1150]
Currently, even if every output file of the main pipeline is not changed between multiple reruns, when we archive coverage maps, the archives themselves will be different in terms of their binary contents.

This is likely due to the fact that our archival program, tar, is capturing timestamps (ctime, mtime, atime) and stores them in the archive.

As the result, it is always true that Kive reports a change between reruns, even if they produced identical results.

For the fix, we should ensure that `tar` behaves time-independently.

**Comment by donkirkby** [comment]
Another option is to deploy the next release of Kive, so we can have output directories, instead of only output files. That way, we won't need any tar or zip files.

I can't remember if we also had a problem with the PDF files' time stamps, or if we used a constant timestamp to avoid the problem.

**Comment by Donaim** [comment]
@donkirkby yes, PDF files have the same issue. See `container_runs/1408362/`
